content/en/blog/2021/direct-upgrade/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Announcing Support for 1.8 to 1.10 Direct Upgrades"
+description: "Moving Towards a Smoother Upgrade Process."
+publishdate: 2021-05-24
+attribution: "Mitch Connors (Google), Sam Naser (Google)"
+keywords: [upgrade,Istio,revision]
+---
+
+As Service Mesh technology moves from cutting edge to stable infrastructure many users have expressed an interest in upgrading their service mesh less frequently, as qualifying a new minor release can take a lot of time.  This can be especially difficult for users who don’t keep up with new releases, as Istio has not supported upgrades across multiple minor versions.  To upgrade from `1.6.x` to `1.8.x`, users first had to upgrade to `1.7.x`.
+
+With the release of Istio 1.10, we are announcing (Alpha|Experimental) level support for upgrading directly from Istio `1.8.x` to `1.10.x`, without upgrading to `1.9.x`.  We hope this will reduce the operational burden of running Istio, in keeping with our 2021 theme of improving Day 2 User Experience.
+
+## Upgrade From 1.8 to 1.10
+
+For direct upgrades we recommend using the canary upgrade method so that control plane functionality can be verified before cutting workloads over to the new version. We’ll also be using revision tags in this guide, an improvement to canary upgrades that was introduced in 1.10 to allow users to label their namespaces once with stable identifiers.
+
+First, create a revision tag stable pointed to the 1.8 revision (from here on out let’s assume this revision is called `1-8-5`). To do this, run:
+
+{{< text bash >}}
+$ istioctl110 x revision tag set stable --revision 1-8-5
+{{< /text >}}
+
+If your 1.8 installation did not have an associated revision, we can create this revision tag with:
+
+{{< text bash >}}
+$ istioctl x revision tag set stable --revision default
+{{< /text >}}
+
+Now, relabel your namespaces that were previously labeled with `istio-injection=enabled` or `istio.io/rev=<REVISION>` with `istio.io/rev=stable`. Download the Istio 1.10.0 release and install the new control plane with a revision:
+
+{{< text bash >}}
+$ istioctl110 install --revision 1-10-0 -y
```

Comment:
again with istioctl110

Reply:
Also, if this was part of the 1.10 download, it was downloaded after you used the command previously.